ui/src/app/views/workspace/component.html

```diff
@@ -5,26 +5,34 @@
 </div>
 <div class="workspace-container">
   <div class="main-body">
-    <h2 class="greeting">{{showTip ? 'Get Started' : 'Recent Work'}}</h2>
-    <div style="width: 200px">
-      <div role="tooltip" aria-haspopup="true" class="tooltip tooltip-md tooltip-top-right">
-        <span *ngIf="!writePermission" class="tooltip-content">Write permission required to create cohorts</span>
-        <div class="card add-card" [ngClass]="{'disabled' : !writePermission}" (click)="!writePermission || buildCohort()">
-          <div class="create-text">
-            Create a <br/>New Cohort
-            <clr-icon shape="plus-circle" class="plus-circle"></clr-icon>
+    <div style="display: flex; flex-direction: row">
+      <h2 class="greeting" style="width: 200px">Actions</h2>
+      <h2 class="greeting" style="padding-left: 10%; margin-left:2rem">Recently Modified Items</h2>
+    </div>
+    <div style="display: flex; flex-direction: row; align-items: center">
+      <div style="width: 200px;">
+        <div role="tooltip" aria-haspopup="true" class="tooltip tooltip-md tooltip-top-right">
+          <span *ngIf="!writePermission" class="tooltip-content">Write permission required to create cohorts</span>
+          <div class="card add-card" [ngClass]="{'disabled' : !writePermission}" (click)="!writePermission || buildCohort()">
+            <div class="create-text">
+              Create a <br/>New Cohort
+              <clr-icon shape="plus-circle" class="plus-circle"></clr-icon>
+            </div>
           </div>
         </div>
-      </div>
-      <div role="tooltip" aria-haspopup="true" class="tooltip tooltip-md tooltip-top-right">
-        <span *ngIf="!writePermission" class="tooltip-content">Write permission required to create notebooks</span>
-        <div class="card add-card" [ngClass]="{'disabled' : !writePermission }" style="margin-top: 13px;" (click)="!writePermission || newNotebook()">
-          <div class="create-text">
-            Create a<br/>New Notebook
-            <clr-icon shape="plus-circle" class="plus-circle"></clr-icon>
+        <div role="tooltip" aria-haspopup="true" class="tooltip tooltip-md tooltip-top-right">
+          <span *ngIf="!writePermission" class="tooltip-content">Write permission required to create notebooks</span>
+          <div class="card add-card" [ngClass]="{'disabled' : !writePermission }" style="margin-top: 13px;" (click)="!writePermission || newNotebook()">
+            <div class="create-text">
+              Create a<br/>New Notebook
+              <clr-icon shape="plus-circle" class="plus-circle"></clr-icon>
+            </div>
           </div>
         </div>
       </div>
+      <div style="padding-left: 10%">
+        <app-recent-work [workspace]="workspace"></app-recent-work>
```

Comment:
FYI(no action needed): It almost feels like the it would be cleaner if the input were a list of RecentResource, but since this scrolling component could potentially lazily fetch more data in the future, I think the approach you have is probably reasonable. Alternatively it could be some generic interface which gives you a list of RecentResources and allows you to request more.